.github/workflows/test_local_integration.yaml

```diff
@@ -34,14 +34,16 @@ on:
         required: true
         type: string
 
+concurrency:
```

Comment:
Can you add a line here to explain what's the expected behaviour?

Reply:
@aktech let me know if this helps